schema/store_item.go

```diff
@@ -15,3 +15,31 @@ type StoreItem struct {
 	Annotations            map[string]string `json:"annotations"`
 	ReadOnlyRootFilesystem bool              `json:"readOnlyRootFilesystem"`
 }
+
+//StoreFunction represents a multi-arch function in the store
+type StoreFunction struct {
```

Comment:
Broader conversation here, @LucasRoesler @stefanprodan would this be useful in faas-provider?

Reply:
I think StoreFunction could embed the Function type like:

```go
import "github.com/openfaas/faas-provider/v1"

type StoreFunction struct {
	v1.Function
	Icon                   string            `json:"icon"`
	RepoURL          string            `json:"repo_url"`
}
```

Reply:
The primary benefit I can see of including it in the faas-provider would be for exporting/reconstructing the definition.  Is this important that all providers can return the original store function definition?

Reply:
Thanks for the input, let's come back to this after the PR is merged?